src/fabric.c

```diff
@@ -650,6 +650,7 @@ void fi_ini(void)
 	ofi_pmem_init();
 	ofi_perf_init();
 	ofi_hook_init();
+	ofi_hmem_init();
```

Comment:
Is there a missing ofi_hmem_cleanup() that pairs with this?

Reply:
The ofi_hmem_cleanup() already exists.